grouper/models/user.py

```diff
@@ -0,0 +1,82 @@
+import logging
+
+from sqlalchemy import Boolean, Column, Integer, String
+from sqlalchemy.ext.hybrid import hybrid_property
+from sqlalchemy.orm import relationship
+
+from grouper.constants import GROUP_ADMIN, MAX_NAME_LENGTH, PERMISSION_ADMIN, USER_ADMIN
+from grouper.models.audit_log import AuditLog
+from grouper.models.base.model_base import Model
+from grouper.models.comment import CommentObjectMixin
+from grouper.models.counter import Counter
+from grouper.models.user_token import UserToken
+from grouper.plugin import get_plugins
```

Comment:
So at the moment I've left 3 of the models imports as is: Model, which must  be imported because it's the base model, CommentObjectMixin, because User must inherit from it, and Counter, because that's needed in User.add which is a method inherited from Model, so moving it out to another module would require changing that method for every other model in Grouper.